scalajslib/src/mill/scalajslib/ScalaJSModule.scala

```diff
@@ -146,12 +146,19 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
   override def runLocal(args: Task[Args] = T.task(Args())): Command[Unit] = T.command { run(args) }
 
   override def run(args: Task[Args] = T.task(Args())): Command[Unit] = T.command {
+    val jsEnvConfigWithArgs = jsEnvConfig() match {
+      case c: JsEnvConfig.NodeJs => c.copy(args = args().value.toList)
+      case c: JsEnvConfig.ExoegoJsDomNodeJs => c.copy(args = args().value.toList)
+      case c: JsEnvConfig.JsDom => c.copy(args = args().value.toList)
+      case c: JsEnvConfig.Phantom => c.copy(args = args().value.toList)
+      case c => c
```

Comment:
We should use `T.log` to let the user know in case we match some unknown type and args is non-empty.

Reply:
Done ✅